delta/rdf/src/test/scala/ch/epfl/bluebrain/nexus/delta/rdf/jsonld/ExpandedJsonLdSpec.scala

```diff
@@ -24,6 +27,12 @@ class ExpandedJsonLdSpec extends AnyWordSpecLike with Matchers with Fixtures {
       ExpandedJsonLd(compacted).accepted shouldEqual ExpandedJsonLd.expanded(expectedExpanded).rightValue
     }
 
+    "be constructed successfully with a base defined in JsonLdOptions" in {
```

Comment:
I'm not sure this needs to be tested at all, since it is part of the json-ld spec and the underlying library. But well I guess it doesn't hurt